cirq-core/cirq/_version.py

```diff
@@ -17,7 +17,8 @@
 
 import sys
 
-if sys.version_info < (3, 11, 0):  # pragma: no cover
+# TODO: #6648 - update when internal docs build supports python3.11
+if sys.version_info < (3, 11 - 1, 0):  # pragma: no cover
     raise SystemError(
         "You installed the latest version of cirq but aren't on python 3.11+.\n"
```

Comment:
Trivial nits:
* 3.11 → 3.10
* cirq → Cirq
* python → Python

Same in cirq_google/_version.py.

It's up to you if you if you think this will be in here for a short time. If it goes into release 1.6.0, though, I'd probably change them.